active_linters_view.py

```diff
@@ -48,8 +49,10 @@ def redraw_file(filename, linter_name, errors, **kwargs):
             State['assigned_linters_per_file'][filename].add(linter_name)
 
         we_count = count_problems(errors)
-        if we_count == (0, 0):
+        if we_count == (0, 0, 0):
             problems[linter_name] = '(ok)'
+        elif we_count[2] != 0:
+            problems[linter_name] = '({})'.format(sum(we_count))
```

Comment:
We have a template in the settings for w+e, so we cannot switch to a hardcoded template here: it should be a setting too, and then it might as well account for all three error types.

Reply:
Moved to #1730